src/ui/server/controllers/insurance/business/map-and-save/index.ts

```diff
@@ -39,6 +40,43 @@ const companyDetails = async (formBody: RequestBody, application: Application, v
   }
 };
 
+/**
+ * maps nature of business request and calls save function
+ * returns true or false based on response from save function
+ * @param {RequestBody} formBody
+ * @param {Application} application
+ * @param {ValidationErrors} validationErrors
```

Comment:
I don't think Application and ValidationErrors would compile into actual documentation(?), should we use `Object`?